File: src/main/java/com/edd/jelly/Launcher.java
package com.edd.jelly;

import com.badlogic.gdx.ApplicationAdapter;
import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

public class Launcher extends ApplicationAdapter {

    public static void main(String... args) {
        LwjglApplicationConfiguration configuration =
                new LwjglApplicationConfiguration();

        configuration.foregroundFPS = 60;
        configuration.width = 800;
        configuration.height = 600;

        configuration.fullscreen = false;
        configuration.resizable = false;

        new LwjglApplication(new LiquidFunTest(), configuration);
    }

    @Override
    public void render() {
        System.out.println("running, dt: " + Gdx.graphics.getDeltaTime());
    }
}

File: src/main/java/com/edd/jelly/Launcher.kt
package com.edd.jelly

import com.badlogic.gdx.ApplicationAdapter
import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.edd.jelly.core.Game
import com.edd.jelly.util.Configuration

class Launcher : ApplicationAdapter() {

    companion object {

        @JvmStatic
        fun main(args: Array<String>) {

            val configuration = LwjglApplicationConfiguration()

            configuration.foregroundFPS = Configuration.FPS_LIMIT
            configuration.width = Configuration.SCREEN_WIDTH
            configuration.height = Configuration.SCREEN_HEIGHT

            configuration.fullscreen = Configuration.FULLSCREEN
            configuration.resizable = Configuration.RESIZABLE

            LwjglApplication(Game(), configuration)
        }
    }
}